src/components/NcRichContenteditable/NcAutoCompleteResult.vue

```diff
@@ -127,21 +129,22 @@ export default {
 	}
 
 	&__status {
+		--auto-complete-result-status-icon-size: clamp(14px, var(--auto-complete-result-avatar-size) * 0.4, 18px);
+		// 2 * Avatar Radius * (1 - 1 / sqrt(2)) - Status Icon Size / 2
+		--auto-complete-result-status-icon-position: calc(var(--auto-complete-result-avatar-size) / 2 * (1 - 1 / sqrt(2)) - var(--auto-complete-result-status-icon-size) / 2);
```

Comment:
![image](https://github.com/user-attachments/assets/72f9193c-42a5-4aea-9a09-63aba4b02046)

Reply:
If it goes well, I'll try to adapt it to NcAvatar and then use NcAvatar here to not duplicate it.